lib/http/index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   RangeParser: require('./range-parser').RangeParser,
-  request: require('./request')
+  request: require('./request'),
+  requestBuilder: require('./request-builder').requestBuilder
```

Comment:
I had thought to put this under util and drop the `http` directory completely.  So there would be 2 different request modules at the same time so that people can easily migrate deprecated requests calls to new ones.